_includes/head.html

```diff
@@ -3,6 +3,11 @@
     <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
     <title>{{ page.title }} - 1418</title>
     <meta name="theme-color" content="#d12727">
+    <meta name="description" content="George Mason High School Robotics Team">
+    <meta name="twitter:card" content="summary_large_image">
+    <meta property="og:title" content="{{ page.title | escape }}">
```

Comment:
Looks good, although I don't know why we need description twice. We could maybe get it from a different place as well?